state/factory/historyfactory.go

```diff
@@ -43,3 +43,8 @@ func (hReader *historyStateReader) States(opts ...protocol.StateOption) (uint64,
 	}
 	return hReader.height, iterator, nil
 }
+
+// ReadView reads the view
+func (hReader *historyStateReader) ReadView(name string) (interface{}, error) {
+	return hReader.sf.ReadView(name)
+}
```

Comment:
wrong implementation?

Reply:
can you be more specific what is wrong?

Reply:
for now historyfactory.go seems just calling sf.Method() so just follow suite, do u mean for historyfactory the view is different for different height?

Reply:
You can leave it as `return errors.New("not implemented")`

Reply:
fixed in latest push